### PR TITLE
Item maintenance

### LINKED ISSUE #7: Item controller
Controller to add, remove, and edit items.

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
Closes #7

### CODE REVIEW

**Comment by josephbridgwaterrowe** [comment]
@michaelachrisco please review.

File: spec/controllers/fagus/items_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      it_behaves_like 'an edit request'
+      it { expect(assigns(:item)).to be_kind_of(Item) }
+    end
+
+    describe 'GET index' do
+      before do
+        allow(controller).to receive(:items) { items }
+        get :index
+      end
+      let(:items) { [ build_stubbed(:item), build_stubbed(:item) ] }
```
Comment (michaelachrisco) [inline spec/controllers/fagus/items_controller_spec.rb]: I believe you can use Factory Girl's`build_list` here to stub out the items.

File: app/helpers/fagus/items_helper.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module Fagus
```
Comment (michaelachrisco) [inline app/helpers/fagus/items_helper.rb]: You will need to remove this. Do you automate the creation of your controllers?

File: spec/controllers/fagus/items_controller_spec.rb (hunk 1 above)
Comment (josephbridgwaterrowe) [inline spec/controllers/fagus/items_controller_spec.rb]: Even better, `build_stubbed_list`

File: app/helpers/fagus/items_helper.rb (hunk 2 above)
Comment (josephbridgwaterrowe) [inline app/helpers/fagus/items_helper.rb]: No, `rails g controller` FTW.

File: spec/controllers/fagus/items_controller_spec.rb (hunk 1 above)
Comment (michaelachrisco) [inline spec/controllers/fagus/items_controller_spec.rb]: :+1:

**Comment by michaelachrisco** [comment]
Rebase and I will merge. Looks good! 

BTW you may want to change the Pull request description to #7. It is already going to close itself once I merge.

**Comment by josephbridgwaterrowe** [comment]
Whoops :smile:

Thanks.

**Pull request merged**